README.md

```diff
@@ -42,19 +43,35 @@ jupyter labextension list
 
 ### Development install
 
-Note: You will need NodeJS to build the extension package.
+It is recommended to first install JLNE and all of its dependencies on a fresh environment to ensure it is stable.
+
+1) Start by installing ![jupyterlab](https://jupyter.org/install) and all of its dependencies.
+2) Install ipykernel
+```
+python -m ipykernel install --name conda_env --user
+```
+3) Clone the repo to your local environment and change the working directory to the jupyterlab_nodeeditor directory.
+4) Install the package via pip and copy package.json over
+```
+pip install -e .
+cat jupyterlab_nodeeditor/labextension/package.json
+```
+5) Install ![NodeJS](https://nodejs.org/en/download/package-manager/)
+6) Install jlpm and yaml
```

Comment:
make pyyaml part of the conda installation, and you aren't installing `jlpm` you are using it to install the javascript deps